src/utils/OneSignalShimLoader.ts

```diff
@@ -20,12 +20,21 @@ export class OneSignalShimLoader {
 
   // Some logic from SdkEnvironment
   private static getPathAndPrefix(): string {
-    if (typeof __DEV__ !== "undefined" && __DEV__) {
-      return "https://localhost:3001/sdks/Dev-";
-    } else if (typeof __STAGING__ !== "undefined" && __STAGING__) {
-      return `https://${window.location.host}/sdks/Staging-`;
+    const buildOrigin = (typeof __BUILD_ORIGIN__ !== "undefined") ? __BUILD_ORIGIN__ || "localhost" : "localhost";
+    const productionOrigin = "https://cdn.onesignal.com/sdks/";
+
+    if (typeof __BUILD_TYPE__ === "undefined") {
```

Comment:
This if is not needed, since we account for it under the `switch` below as `default:`

Reply:
if __BUILD_TYPE__ is undefined, the switch breaks